[README] OCaml package `ppx_deriving_yojson` is missing

Hello,

While running `make` on a new virtual machine, I got a failure about `ppx_deriving_yojson`:

```text
$ make
Checking the charon installation
cd src && dune build
File "dune", line 40, characters 2-21:
40 |   ppx_deriving_yojson)
       ^^^^^^^^^^^^^^^^^^^
Error: Library "ppx_deriving_yojson" not found.
-> required by library "aeneas" in _build/default
-> required by _build/default/META.aeneas
-> required by alias all
-> required by alias default
File "dune", line 30, characters 3-22:
30 |    ppx_deriving_yojson
        ^^^^^^^^^^^^^^^^^^^
Error: Library "ppx_deriving_yojson" not found.
-> required by _build/default/BorrowCheck.pp.ml
-> required by alias all
-> required by alias default
make: *** [Makefile:65: build-bin] Error 1         
```

This is easy to fix with `opam install ppx_deriving_yojson`, and this package was not installed on my system because this dependency was not listed in `README.md`: https://github.com/AeneasVerif/aeneas/blob/bcfa1ca15f5b5cb4c3198a1db1e4802a82bf1a68/README.md#L48-L52

This new dependency was actually introduced last week with https://github.com/AeneasVerif/aeneas/pull/1009 , which forgot to update the installation instructions in `README.md`. I do not know whether this list is automatically generated in some way (from [`flake.nix`](https://github.com/AeneasVerif/aeneas/blob/bcfa1ca15f5b5cb4c3198a1db1e4802a82bf1a68/flake.nix)?), which is why I chose to open a GitHub issue first. If you want, I can submit a Pull Request.

--- a/README.md
+++ b/README.md
@@ -47,8 +47,9 @@ opam switch create 5.3.0
 
 You can then install the dependencies with the following command:
 ```
-opam install ppx_deriving visitors easy_logging zarith yojson core_unix odoc \
-  ocamlgraph menhir ocamlformat.0.27.0 unionFind zarith progress domainslib
+opam install calendar core_unix domainslib easy_logging menhir \
+  ocamlformat.0.27.0 ocamlgraph odoc ppx_deriving ppx_deriving_yojson \
+  progress unionFind visitors yojson zarith
 ```
 
 Moreover, Aeneas uses the [Charon](https://github.com/AeneasVerif/charon) project and library.

